### PR TITLE
Remove the only use of EnforceConsumed

### PR DESCRIPTION
**Pull request by houseroad**
Remove the EnforceConsumed in BatchNorm spec.

### CODE REVIEW

**Comment by houseroad** [comment]
@onnxbot retest this please

**Comment by tjingrant** [comment]
EnforceConsumed seems to be in the pip release as well. This is causing CI troubles for us : https://travis-ci.org/onnx/onnx-tensorflow/jobs/356599866. Please update the pip package as well.

**Comment by houseroad** [comment]
@tjingrant I think you should be good now.

**Comment by linkerzhang** [comment]
so this is good to be merged now? :)
